modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsMapFieldPropertyValue.php

```diff
@@ -23,7 +23,8 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
     $key = $this->configuration['key'];
     $value_key = $this->configuration['value'];
     // Get the field values.
-    if ($key === 'body_format') {
+
+    if ($key === 'body_format' || $key === 'options') {
       $value[$key] = $value_key;
     }
```

Comment:
It's not currently documented that the plugin behaves differently for these special key names. Perhaps tie this behavior to the existence of an optional  `$this->configuration['default_value'];` to say the value is being mapped as a default rather than as a field and remove these special case keys.

This would make it easier to tell in the migration.yml that the value is being treated this way rather than mapped as a field.

Reply:
@tadean  - For the `field_az_accordion`, the property name little different as `body_format` that is why it mentioned as it. Yes you are right about the field name, we can remove it from iml , because it is not using.

Reply:
As per the comment changes are done.